### PR TITLE
revamped detail page

### PR DESCRIPTION
**Pull request by cnnebe**
made extensive changes to the layout of the action idea detail page. Also made some minor changes to some layouts across site.

### CODE REVIEW

File: HandprintGenerator/templates/HandprintGenerator/detail.html [hunk 1]
```diff
@@ -88,31 +105,85 @@ <h4 class="text-center"> Comments </h4>
...
+                        <hr>
+                        <h4 data-toggle="tooltip" title="Sources of inspiration or further reading."> <u>References</u> <br>{{ ai.references }}</h4>
+                        <br>
+                        {% endif %}
+                    <!-- /.input-group -->
+                </div>
+
+
+                <!-- Side Widget Well -->
+                <div class="well" id="details">
```
Comment (jinjennifer) [inline HandprintGenerator/templates/HandprintGenerator/detail.html]: random empty side well if there are no tags

**Comment by jinjennifer** [comment]
I like the general styling of the detail page and it looks much better. Do you think it would be a good idea to move everything (votes, description, vote button, edit/delete action idea button) into the side well since the photo is so large? There's a lot of white space below the category and tags. I also think there may be an issue if they upload a low res photo - the image is blown up and very pixelated. Should we adjust the size of the photo and let the well resize? Or just have white space around the photo (that may be the better idea)?

Also thanks for fixing the styling issue with creating a new action idea :) Do you think you can copy that styling over to the edit action idea form?

**Comment by cnnebe** [comment]
Yeah, I could move them all to the side wells. And only have the comments, pictures, and descriptions on the left. I think it would make it look a little better. And, I can make it so that there is a max size for the photo. So, if the image is small. It won't have to be  pixelated. There will just be a lot of white space, for some images. But, that might not be a problem if its formatted well. 

And no problem, I'll fix the edit action form later today.

**Comment by jinjennifer** [comment]
Also, since users don't have images, should the image for the user in the comment section be removed? Could the report and delete comment buttons be moved to the right of the comment? I feel like the buttons take up unnecessary space.

**Comment by cnnebe** [comment]
I made the changes you wanted. I couldn't figure out the delete and report button under comments though. Whatever I did, it ended up looking weird.

**Comment by jinjennifer** [comment]
Thanks for making the changes. Can you also change the edit action idea form to look like the new action idea form? Could you also put everything one box in the side well? I feel like multiple boxes is distracting it. 

Also, how do you feel about the background in the view all page? Weinberg suggested a different background/the background only shows up in view all, not with the different filters which can be sort of jarring.

**Comment by cnnebe** [comment]
Oh sorry about that, I made changes to the delete action idea form. For some reason I couldn't find the edit action idea form, and, I had mean't to ask you about that. I'll look again though. And yeah, I'll add them to the same box and work on making the side bar look better.

And, I had mean't to ask you guys about that. Personally, I liked it. But, if you guys don't think it works then I can change it. I'm not sure how good it would look on other pages though. Like, I don't think It would look very good on the home page.

**Comment by jinjennifer** [comment]
Yeah one thing that Weinberg said is that it feels like there are multiple sites, since the look/feel of them are kind of different and the background sort of contributes to that. I think it would be easiest to standardize it to the simplest background, but I want to see what @mcvellaa and @jramjohn think.

**Pull request merged**